agent/src/beerocks/fronthaul_manager/ap_manager/ap_manager_thread.h

```diff
@@ -87,10 +87,7 @@ class ap_manager_thread : public beerocks::socket_thread {
     std::string m_iface;
     beerocks::logging &m_logger;
     bool m_ap_manager_configured = false;
-    uint8_t wifi_channel;
-    uint8_t m_iface_type;
     bool acs_enabled;
-    bool low_filter;
```

Comment:
This shouldn't be a fixup commit but a real separate commit.

Reply:
You will forgive me for not adding more commits to this PR, right? :)